docs/containers/bridge-to-kubernetes.md

```diff
@@ -20,141 +20,67 @@ In this guide, you will learn how to use Bridge to Kubernetes to redirect traffi
-
-All traffic in the Kubernetes cluster is redirected for the bikes service to the version of your application running in your development computer. Bridge to Kubernetes also routes all outbound traffic from the application back to your Kubernetes cluster.
+All traffic in the Kubernetes cluster is redirected for your service to the version of your application running in your development computer. Bridge to Kubernetes also routes all outbound traffic from the application back to your Kubernetes cluster.
 
 > **Important**: You can only redirect services that have a single pod.
 
-After you select your service, you are prompted to enter the TCP port for your local application. For this example, enter "3000".
+After you select your service, you are prompted to enter the TCP port that your application uses to run locally.
 
-![Connect choose port](images/bridge-to-kubernetes-vs-code/choose-port.png)
+![Enter the port number](images/minikube/enter_port.png)
 
-Choose **Launch via NPM** as the launch task.
+Choose a debug profile that you normally use when running your application locally.
 
-![Connect choose launch task](images/bridge-to-kubernetes-vs-code/choose-launch.png)
+![Choose the debugger launch task](images/minikube/launch_task.png)
 
 > **Note**: You will be prompted to allow the **EndpointManager** to run elevated and modify your hosts file.
 
 You have the option of running isolated or not isolated. If you run isolated, only your requests are routed to your local process; other developers can use the cluster without being affected. If you don't run isolated, all traffic is redirected to your local process. For more information on this option, see [Using routing capabilities for developing in isolation][btk-overview-routing].
 
-![Isolation prompt](images/bridge-to-kubernetes-vs-code/btk-isolation-prompt.png)
+![Choose isolation](images/minikube/isolation.png)
+
+Select the **Debug** icon on the left and select **Launch via NPM with Kubernetes** at the top. This debug profile is created by Bridge to Kubernetes from the debug profile you chose earlier.
 
-Select the **Debug** icon on the left and select **Launch via NPM with Kubernetes** at the top.
+![Choose debug launch profile](images/minikube/debug_profile.png)
 
-![Choose Bridge to Kubernetes](images/bridge-to-kubernetes-vs-code/choose-bridge-to-kubernetes.png)
+> **Note**: You can open *.vscode/launch.json* to see the specific configuration settings that Bridge to Kubernetes adds to your launch profile. If your cluster uses [gRPC C core](https://github.com/grpc/grpc/), an implementation of gRPC that uses [c-ares](https://github.com/c-ares/c-ares), an environment variable is added to your launch profile, GRPC_DNS_RESOLVER, with the value "native." This variable specifies to use a workaround to avoid a 2-minute time delay when connecting. For more information, see this [gRPC issue #18691](https://github.com/grpc/grpc/issues/18691).
```

Comment:
Minor comment: I'd suggest to say ` You can open *.vscode/launch.json*/*.vscode/tasks.json*` as part of our configuration is in tasks.json.